L/LibSerialPort/Compat.toml

```diff
@@ -0,0 +1,2 @@
+["0.1"]
+julia = "0.7-1.1"
```

Comment:
This is a bug https://github.com/JuliaComputing/Registrator.jl/issues/103 so please retrigger the bot when the linked issue is fixed.